pkg/autoscaler/autoscaler_test.go

```diff
@@ -113,3 +114,141 @@ func TestAutoscaler_Awake(t *testing.T) {
 		return latestRequest != 1
 	}, downscaleStabilizationPeriod, time.Second)
 }
+
+func TestAutoscaler_MinReplicas(t *testing.T) {
+	t.Parallel()
+	log := newLogger(t)
+
+	mux := sync.RWMutex{}
+	var latestRequest int32
+
+	minReplicas := int32(5)
+	maxReplicas := int32(10)
+
+	scalerMock := &ScalerFunc{
+		ScaleFunc: func(apiName string, request int32) error {
+			mux.Lock()
+			defer mux.Unlock()
+
+			latestRequest = request
+			return nil
+		},
+		GetInFlightRequestsFunc: func(apiName string, window time.Duration) (*float64, error) {
+			return pointer.Float64(0), nil
+		},
+		GetAutoscalingSpecFunc: func(apiName string) (*userconfig.Autoscaling, error) {
+			return &userconfig.Autoscaling{
+				MinReplicas:    minReplicas,
+				MaxReplicas:    maxReplicas,
+				InitReplicas:   minReplicas,
+				TargetInFlight: pointer.Float64(1),
+				Window:         time.Second,
+			}, nil
+		},
+		CurrentReplicasFunc: func(apiName string) (int32, error) {
+			return minReplicas + 1, nil
```

Comment:
Where does the `+1` come from?

Reply:
This is to make sure that the value is different from the minReplicas value. This way a scale operation will be triggered